packages/@okta/i18n/src/properties/login_zh_CN.properties

```diff
@@ -1358,9 +1358,9 @@ idx.return.link.otponly.app = {0}
 # {0} is the name of the browser and {1} is the name of the devices OS that triggered the login
 idx.return.link.otponly.browser.on.os = {1} \u5728 {0}
 # {0} is the city, {1} is the state, and {2} is the country from where the login request originated
-geolocation.formatting.all = {0}, {1}, {2}
+geolocation.formatting.all = {2}{1}{0}
```

Comment:
This should follow the `zh_TW` file which is 

` {0}\uFF0C{1}\uFF0C{2}` 

cc: @pengyao-okta

Reply:
To me, I think the {2}, {1}, {0} or {2}{1}{0} is correct for CN. If having UI screenshot could help to distinguish.

Reply:
I already send msg to vendor to confirm.

Reply:
I will followup with this on slack for the ss :).